src/components/container/RestingHeartRateCalendar/RestingHeartRateCalendar.tsx

```diff
@@ -10,12 +10,14 @@ import { queryDailyData, DailyDataType } from '../../../helpers/query-daily-data
 
 type HeartRateMap = { [key: string]: number };
 
-export type RestingHeartRateCalendarPreviewState = "WithData" | "NoData" | "Loading";
+export type RestingHeartRateCalendarPreviewState = "WithData" | "NoData" | "Loading" | "Live";
+export type RestingHeartRateDataSource = "Combined" | "AppleHealth" | "Fitbit" | "Garmin";
 
 export interface RestingHeartRateCalendarProps {
 	month: number,
 	year: number,
 	showPreviewData: RestingHeartRateCalendarPreviewState
+	dataTypeSource?: RestingHeartRateDataSource
```

Comment:
Advocating for 
1) RestingHeartRateDataSource to have Combined removed
2) The RestingHeartRateCalendarProps.dataTypeSource to be an array of RestingHeartRateDataSource 

This will allow for supporting a subset of devices instead of all/one

Reply:
I am curious about the use case for supporting this.  This data provider wants to return all available data.

Reply:
See here for an example https://github.com/CareEvolution/MyDataHelpsUI/blob/56f41e637154b522b43a888119e49fb86404cc69/src/components/container/BloodPressureVisualization/BloodPressureVisualization.tsx#L21

Reply:
The current framework for daily data providers is  straightforward, returning data without specifying options.  I think there is a future opportunity to create a new type of data provider, that would allow you to specify specific data sources that are rolled up together.  However, for this current work, returning all available data is what is expected.  I will open a new issue about generalizing this type of combined data provider.